README.md

```diff
@@ -420,7 +420,7 @@ Result:
 
 3. Create a project
    ```
-   kalix projects new ecommerce --region=gcp-us-east1
+   kalix projects new ecommerce --region=gcp-us-east1 --organization=my-organization
```

Comment:
It might be nice to add a step above this one suggesting the user run `kalix organization list` so they know what organization to provide in this step.  (It should have the same as their username...)  Just so they don't sit there and try with `my-organization` and wonder why it's not working.

Reply:
I added a step. Can you give this another look and let me know if it's more clear?